docs/developer-docs/integrations/sns/tokenomics/sns-checklist.md

```diff
@@ -57,46 +58,28 @@ It may not be possible to fully decentralize the dapp at the time of the SNS lau
-
-<!-- NEW: As a part of the SNS launch process, an NNS proposals must be created. You can find the details about how to create such a proposal [here](../launching/launch-summary-1proposal.md).-->
-
-
-### 1.5.1. SNS canister creation proposal
-The canister creation proposal is asking the NNS neurons for acceptance to start the SNS launch process, and to allow only the accepted principal to install an SNS.
+<!-- OLD: As a part of the SNS launch process, two proposals must be created. The first proposal is for allowing a principle to create the SNS canisters, see Hot or Not’s [command](https://forum.dfinity.org/t/help-submit-the-sns-w-wallet-principal-id-to-be-whitelisted/20322/20) to create the first proposal.-->
 
-This proposal is not going in-depth with the tokenomics, governance etc. but links to the dapp and a whitepaper should be shared, so the community can evaluate if launching a SNS for the dapp makes sense. See OpenChat’s [proposal 1](https://dashboard.internetcomputer.org/proposal/108634) for inspiration.
+ As a part of the SNS launch process, an NNS proposal must be created to ask the NNS to create an SNS and start a decentralization swap. You can find the details about how to create such a proposal [here](../launching/launch-summary-1proposal.md).
 
-<!-- NEW: REMOVE-->
+Include relevant information like tokenomics (token distribution, governance, decentralization swap), details about the dapp (link to the open sourced code), whitepaper and anything else relevant for the community to make a decisions if an SNS should be launched for the dapp. 
 
-### 1.5.2. Decentralization proposal
-The second proposal is asking the NNS neurons for acceptance to start the decentralization process. If the proposal is adopted, the decentralization of the dapp and the decentralization swap will begin.
 
-Include relevant information like tokenomics (token distribution, governance, decentralization swap), details about the dapp (link to the open sourced code), whitepaper and anything else relevant to the community. See OpenChat’s [decentralization proposal](https://dashboard.internetcomputer.org/proposal/109811) for inspiration.
-
-<!-- NEW: REMOVE-->
-
-### 1.5.3. More information
-See the [documentation](./preparation.md) for details about the process. The documentation page [SNS predeployment considerations](./predeployment-considerations.md) has a list of topics that should be covered in the whitepaper/proposal. See this [forum post](https://forum.dfinity.org/t/dfinitys-voting-on-upcoming-sns-launch-proposals/19543) for information about DFINITY’s voting.
-
-:::info
-A 1-proposal process is planned to be rolled out in August 2023. After this, the SNS launch will only require one NNS proposal.
-:::
-<!-- NEW: REMOVE-->
+### 1.5.1. More information
```

Comment:
Since we now have only one subsection, it makes sense to remove subsection headings completely.

Reply:
done